README.md

```diff
@@ -23,27 +23,45 @@ you will need to have PHP installed to run it.
 Install it with your preferred package management tool
 (for example, on Ubuntu Linux you can run: `sudo apt-get install php`)
 
+The recommended way to install this library is with Composer.
+Composer is a dependency management tool for PHP
+that allows you to declare the dependencies your project needs
+and installs them into your project.
+
 Install Composer by following the instructions [here](https://getcomposer.org/doc/00-intro.md).
 
-Then, download Wikimate, and initialise it by running `composer install`
-(or `composer.bat install` if you're on Windows).
+Then, download Wikimate and initialise it by running:
+
+```sh
+composer require hamstar/Wikimate
+```
+
+(or `composer.bat require hamstar/Wikimate` if you're on Windows).
 
-To use Wikimate within another project, you can add it as a composer dependency
-by adding the following to your `composer.json` file:
+To use Wikimate within another project, you can add it as a Composer dependency
+by adding the following to your existing `composer.json` file:
 
 ```json
-"hamstar/Wikimate": "0.15.0"
+{
+    "require": {
+        "hamstar/Wikimate": "^0.15.0"
+    }
+}
 ```
 
+You can find out more on how to install Composer,
+configure autoloading, and other best-practices for defining dependencies
+at [getcomposer.org](https://getcomposer.org).
+
 ## Usage
 
-In your script file (e.g. `index.php`), include Wikimate's `globals.php` file,
+In your script file (e.g. `index.php`), include the project's `autoload.php` file,
```

Comment:
Why is "the project's `autoload.php`" better than "Wikimate's `autoload.php`"? Am I missing something?

Reply:
A project has only one `autoload.php`. If Wikimate is "merely" a dependency in a bigger project, specifying "Wikimate's" file would be confusing. It is a library after all; as developers we're the only ones using Wikimate as our main project.

Reply:
Ah, gotcha. Thanks for clarifying :)